test/liblll/Compiler.cpp

```diff
@@ -121,6 +122,504 @@ BOOST_AUTO_TEST_CASE(switch_inconsistent_return_count)
+    "{ (asm ORIGIN) }",
+    "{ (asm CALLER) }",
+    "{ (asm CALLVALUE) }",
+    "{ (asm CALLDATALOAD) }",
+    "{ (asm CALLDATASIZE) }",
+    "{ (asm CALLDATACOPY) }",
+    "{ (asm CODESIZE) }",
+    "{ (asm CODECOPY) }",
+    "{ (asm GASPRICE) }",
+    "{ (asm EXTCODESIZE)}",
+    "{ (asm EXTCODECOPY) }",
+    "{ (asm RETURNDATASIZE) }",
+    "{ (asm RETURNDATACOPY) }",
+    "{ (asm BLOCKHASH) }",
+    "{ (asm COINBASE) }",
+    "{ (asm TIMESTAMP) }",
+    "{ (asm NUMBER) }",
+    "{ (asm DIFFICULTY) }",
+    "{ (asm GASLIMIT) }",
+    "{ (asm POP) }",
+    "{ (asm MLOAD) }",
+    "{ (asm MSTORE) }",
+    "{ (asm MSTORE8) }",
+    "{ (asm SLOAD) }",
+    "{ (asm SSTORE) }",
+    "{ (asm JUMP ) }",
+    "{ (asm JUMPI ) }",
+    "{ (asm PC) }",
+    "{ (asm MSIZE) }",
+    "{ (asm GAS) }",
+    "{ (asm JUMPDEST) }",
+    "{ (asm PUSH1) }",
+    "{ (asm PUSH2) }",
+    "{ (asm PUSH3) }",
+    "{ (asm PUSH4) }",
+    "{ (asm PUSH5) }",
+    "{ (asm PUSH6) }",
+    "{ (asm PUSH7) }",
+    "{ (asm PUSH8) }",
+    "{ (asm PUSH9) }",
+    "{ (asm PUSH10) }",
+    "{ (asm PUSH11) }",
+    "{ (asm PUSH12) }",
+    "{ (asm PUSH13) }",
+    "{ (asm PUSH14) }",
+    "{ (asm PUSH15) }",
+    "{ (asm PUSH16) }",
+    "{ (asm PUSH17) }",
+    "{ (asm PUSH18) }",
+    "{ (asm PUSH19) }",
+    "{ (asm PUSH20) }",
+    "{ (asm PUSH21) }",
+    "{ (asm PUSH22) }",
+    "{ (asm PUSH23) }",
+    "{ (asm PUSH24) }",
+    "{ (asm PUSH25) }",
+    "{ (asm PUSH26) }",
+    "{ (asm PUSH27) }",
+    "{ (asm PUSH28) }",
+    "{ (asm PUSH29) }",
+    "{ (asm PUSH30) }",
+    "{ (asm PUSH31) }",
+    "{ (asm PUSH32) }",
+    "{ (asm DUP1) }",
+    "{ (asm DUP2) }",
+    "{ (asm DUP3) }",
+    "{ (asm DUP4) }",
+    "{ (asm DUP5) }",
+    "{ (asm DUP6) }",
+    "{ (asm DUP7) }",
+    "{ (asm DUP8) }",
+    "{ (asm DUP9) }",
+    "{ (asm DUP10) }",
+    "{ (asm DUP11) }",
+    "{ (asm DUP12) }",
+    "{ (asm DUP13) }",
+    "{ (asm DUP14) }",
+    "{ (asm DUP15) }",
+    "{ (asm DUP16) }",
+    "{ (asm SWAP1) }",
+    "{ (asm SWAP2) }",
+    "{ (asm SWAP3) }",
+    "{ (asm SWAP4) }",
+    "{ (asm SWAP5) }",
+    "{ (asm SWAP6) }",
+    "{ (asm SWAP7) }",
+    "{ (asm SWAP8) }",
+    "{ (asm SWAP9) }",
+    "{ (asm SWAP10) }",
+    "{ (asm SWAP11) }",
+    "{ (asm SWAP12) }",
+    "{ (asm SWAP13) }",
+    "{ (asm SWAP14) }",
+    "{ (asm SWAP15) }",
+    "{ (asm SWAP16) }",
+    "{ (asm LOG0) }",
+    "{ (asm LOG1) }",
+    "{ (asm LOG2) }",
+    "{ (asm LOG3) }",
+    "{ (asm LOG4) }",
+    "{ (asm CREATE) }",
+    "{ (asm CALL) }",
+    "{ (asm CALLCODE) }",
+    "{ (asm RETURN) }",
+    "{ (asm DELEGATECALL) }",
+    "{ (asm STATICCALL) }",
+    "{ (asm REVERT) }",
+    "{ (asm INVALID) }",
+    "{ (asm SELFDESTRUCT) }",
+  };
+
+  for (size_t i = 0; i < opcodes_bytecode.size(); i++) {
+    std::vector<std::string> errors;
+    bytes code = eth::compileLLL(opcodes_lll[i], false, &errors);
+
+    if (!errors.empty()) {
+      BOOST_FAIL(opcodes_lll[i]);
+    }
```

Comment:
I think this could be replaced with `BOOST_REQUIRE_MESSAGE(errors.empty(), opcodes_lll[i]);` though I'm not 100% sure.

Reply:
done.  and yeah, it works.